client/my-sites/upgrades/components/domain-warnings/index.jsx

```diff
@@ -373,30 +373,50 @@ export default React.createClass( {
 
 	unverifiedDomainsCanManage() {
 		const domains = this.getDomains().filter( domain => domain.isPendingIcannVerification && domain.currentUserCanManage );
+		const severity = domains.some( ( { registrationMoment } ) =>
```

Comment:
I second @omarjackman's suggestion - the line is long and complex and should be split a bit. Maybe even just the predicate?
```
const withinTwoDays = ( { registrationMoment } ) => {
    if ( ! registrationMoment ) {
       return false;
    }
    return moment( registrationMoment ).add( 2, 'days' ).isAfter( moment() ) );
};
const severity = domains.some( withinTwoDays ) ? 'is-info' : 'is-error';
```